### ISSUE TITLE
URLs in quoted tweets aren't expanded when shown via thread command

### ISSUE THREAD

**Issue by werid** [oysttyer/oysttyer#48]
Hi, 

when using the /thread command, any URLs featured in quoted tweets aren't expanded. they show as t.co links. (but the URL to the quoted tweet is shown in full)

how it appears in feed:

```
p0> [ 2:50:41] <"reportedly> Another look at the bus fire in Yinchuan, China: https://twitter.com/PDChina/status/684187174313586688
p1> [ 2:38:28] (x17) <↑PDChina> #Breaking Moving bus caught fire on road in NW #China's Ningxia on Tue morning, causing 14 de  ad & 32 wounded https://pbs.twimg.com/tweet_video_thumb/CX64TSVWAAArYou.png
```

how it appears when viewing via /thread

```
oysttyer> /th p0 <007                                                                                                          
zz0> [ 2:38:28] (x17) <PDChina> #Breaking Moving bus caught fire on road in NW #China's Ningxia on Tue morning, causing 14 dead & 32 wounded https://t.co/GbdYGfyzvT
zz1> [ 2:50:41] <"reportedly> Another look at the bus fire in Yinchuan, China: https://twitter.com/PDChina/status/684187174313586688
```

This behavior seen on 2.6.0.

**Comment by atomicules** [comment]
Thanks for the report. I'll take a look.

**Comment by IvanSanchez** [comment]
I suggest using https://github.com/oysttyer/oysttyer-deshortify as a workaround in the meantime.

**Comment by atomicules** [comment]
Indeed. That's why I often don't notice these issues. I forget to 
disable when developing, etc.

**Comment by atomicules** [comment]
This is quite an interesting issue. Making progress in the branch I've 
just pushed, but not solved it completely.

**Comment by atomicules** [comment]
I'm back on this now I've closed out #49.

**Comment by atomicules** [comment]
Labelling of this issue may have gone haywire - I'm using ghi from the command line to try to change labels. I didn't mean to remove the in progress label. Definitely on this one next/now.
